File: FlightMobileServer/FlightMobileWeb/Model/Entities/Command.cs
using System.Text.Json.Serialization;
namespace FlightGearServer.Model.Entities
{
    public class Command
    {
        [JsonPropertyName("aileron")]
        public double Aileron { get; set; }

        [JsonPropertyName("rudder")]
        public double Rudder { get; set; }

        [JsonPropertyName("elevator")]
        public double Elevator { get; set; }

        [JsonPropertyName("throttle")]
        public double Throttle { get; set; }

    }
}

File: FlightMobileApp/app/src/main/java/com/example/FlightMobileApp/Command.kt
package com.example.FlightMobileApp

import com.google.gson.annotations.SerializedName

/**
 * Command class.
 * serialized to json format
 */
data class Command (
    @SerializedName("aileron") var aileron: Double,
    @SerializedName("rudder") var rudder: Double,
    @SerializedName("elevator") var elevator: Double,
    @SerializedName("throttle") var throttle: Double
)